Makefile

```diff
@@ -21,29 +23,27 @@ travis:
 	@echo building for travis
 
 	rm -Rf $(DIST_ROOT)
-	@go clean $(GOFLAGS) -i ./...
-	
+	@$(GO) clean $(GOFLAGS) -i ./...
+
 	@cd web/react/ && npm install
 
-	@go build $(GOFLAGS) ./...
+	@$(GO) build $(GOFLAGS) ./...
 
 	@mkdir -p logs
 
-	@go test $(GOFLAGS) -run=$(TESTS) -test.v -test.timeout=180s ./api || exit 1
-	@go test $(GOFLAGS) -run=$(TESTS) -test.v -test.timeout=12s ./model || exit 1
-	@go test $(GOFLAGS) -run=$(TESTS) -test.v -test.timeout=120s ./store || exit 1
-	@go test $(GOFLAGS) -run=$(TESTS) -test.v -test.timeout=120s ./utils || exit 1
-	@go test $(GOFLAGS) -run=$(TESTS) -test.v -test.timeout=120s ./web || exit 1
+	@$(GO) test $(GOFLAGS) -run=$(TESTS) -test.v -test.timeout=180s ./api || exit 1
+	@$(GO) test $(GOFLAGS) -run=$(TESTS) -test.v -test.timeout=12s ./model || exit 1
+	@$(GO) test $(GOFLAGS) -run=$(TESTS) -test.v -test.timeout=120s ./store || exit 1
+	@$(GO) test $(GOFLAGS) -run=$(TESTS) -test.v -test.timeout=120s ./utils || exit 1
+	@$(GO) test $(GOFLAGS) -run=$(TESTS) -test.v -test.timeout=120s ./web || exit 1
 
 build:
-	@go build $(GOFLAGS) ./...
+	@$(GO) build $(GOFLAGS) ./...
 
 install:
 	@go get $(GOFLAGS) github.com/tools/godep
 
-	@if [ $(shell docker ps -a | grep -ci mattermost-mysql) -eq 0 ]; then \
-		echo restoring go libs using godep; \
-		$(GOPATH)/bin/godep restore; \
+	@if [ $(shell docker ps | grep -ci mattermost-mysql) -eq 0 ]; then \
```

Comment:
Oh, good catch. That shouldn't have been changed when I was rebasing for the pull request.